src/Symfony/Component/HttpClient/HttpClientTrait.php

```diff
@@ -197,6 +197,10 @@ private static function mergeDefaultOptions(array $options, array $defaultOption
                 continue;
             }
 
+            if ('auth_ntlm' === $name) {
+                throw new InvalidArgumentException(sprintf('Option "auth_ntlm" is not supported by "%s", try using CurlHttpClient instead.', __CLASS__));
```

Comment:
I would prefer to have CurlHttpClient::class as a parameter of the sprintf message to have the FQCN for easier discovery

Reply:
we could also improve the message and make is suggest to install the curl extension when it's not installed? (with `extension_loaded('curl')`)

Reply:
@nicolas-grekas how about `..., try to install the "curl" extension to use "CurlHttpClient" instead.` if curl is not loaded ?

Reply:
:+1: `try installing the "curl" extension to use "CurlHttpClient" instead.`

Reply:
Done ..
- Without curl extension :
![image](https://user-images.githubusercontent.com/7301643/93109897-e8e3a280-f6b4-11ea-977b-fec83bbf176a.png)
-With curl extension : 
![image](https://user-images.githubusercontent.com/7301643/93109995-06187100-f6b5-11ea-9bdb-1495a8534f8d.png)

Reply:
Thanks!